robot/Cumulus/resources/NPSP.robot

```diff
@@ -327,19 +341,7 @@ Run Donations Batch Process
     Wait For Locator    npsp_settings.status    RLLP_OppContactRollup_BATCH    Completed
     Wait For Locator    npsp_settings.status    RLLP_OppHouseholdRollup_BATCH    Completed
     Wait For Locator    npsp_settings.status    RLLP_OppSoftCreditRollup_BATCH    Completed
-    
-# Choose Frame
-    # [Arguments]    ${frame}
-    # Select Frame    //iframe[contains(@title,'${frame}')]
-    
-# Select Frame with Name
-    # [Arguments]    ${name}
-    # Select Frame    //iframe[contains(@name, '${name}')]
-
-# Select Frame With Title
-    # [Arguments]    ${name}
-    # Select Frame    //iframe[@title= '${name}']    
```

Comment:
Are these keywords no longer required ?

Reply:
yes they are not needed anymore, I have a different keyword for frames now